src/sql/workbench/services/notebook/browser/notebookServiceImpl.ts

```diff
@@ -341,31 +336,31 @@ export class NotebookService extends Disposable implements INotebookService {
 		this.sendNotebookCloseToProvider(editor);
 	}
 
-	listNotebookEditors(): INotebookEditor[] {
-		let editors = [];
-		this._editors.forEach(e => editors.push(e));
-		return editors;
-	}
-
 	findNotebookEditor(notebookUri: URI): INotebookEditor | undefined {
 		if (!notebookUri) {
 			return undefined;
 		}
 		let uriString = notebookUri.toString();
-		let editor = find(this.listNotebookEditors(), n => n.id === uriString);
+		let editor = this.listNotebookEditors().find(n => n.id === uriString);
 		return editor;
 	}
 
 	renameNotebookEditor(oldUri: URI, newUri: URI, currentEditor: INotebookEditor): void {
 		let oldUriKey = oldUri.toString();
 		if (this._editors.has(oldUriKey)) {
 			this._editors.delete(oldUriKey);
-			currentEditor.notebookParams.notebookUri = newUri;
-			this._editors.set(newUri.toString(), currentEditor);
+			currentEditor.notebookParams.notebookUri = newUri; //currentEditor.id gets this value as a string
+			this._editors.set(currentEditor.id, currentEditor);
```

Comment:
INotebookEditor's get id() is implemented as notebookParams.notebookUri.toString(), so this is a no-op change, but since all add, remove find uses editor.id as key for the Map, keeping it consistent here makes the code more readable.